test/e2e/storage/persistent_volumes-local.go

```diff
@@ -326,9 +326,9 @@ var _ = utils.SIGDescribe("PersistentVolumes-local", func() {
 			ginkgo.By("Creating local PVC and PV")
 			createLocalPVCsPVs(ctx, config, []*localTestVolume{testVol}, immediateMode)
 			pod, err := createLocalPod(ctx, config, testVol, nil)
-			framework.ExpectError(err)
+			gomega.Expect(err).To(gomega.MatchError(gomega.ContainSubstring("is not Running")))
 			err = e2epod.WaitTimeoutForPodRunningInNamespace(ctx, config.client, pod.Name, pod.Namespace, f.Timeouts.PodStart)
-			framework.ExpectError(err)
+			gomega.Expect(err).To(gomega.MatchError(framework.ErrFailure))
```

Comment:
`e2epod.WaitTimeoutForPodRunningInNamespace` + `expect error` -> this should be a `gomega.Consistently`.

Checking for `framework.ErrFailure` is too unspecific.

Reply:
Patrick， it looks like "e2epod.WaitTimeoutForPodRunningInNamespace + expect error" could be removed here, due to "pod, err := createLocalPod(ctx, config, testVol, nil) +gomega.Expect(err).To(gomega.MatchError(gomega.ContainSubstring("is not Running")))" has got the pod is not running err.   what do you think about it?

Reply:
I think you are right.

https://github.com/kubernetes/kubernetes/blob/97332c1edca5be0082414d8a030a408f91bed003/test/e2e/framework/pod/create.go#L106-L128

already waits for `config.timeouts.PodStart`. Waiting twice just adds some assurance that it *really* doesn't start, which seems unnecessary.

Let's remove.